lib/box-client.js

```diff
@@ -208,6 +209,7 @@ function BoxClient(apiSession, config, requestManager) {
 	this.devicePins = new DevicePins(this);
 	this.webhooks = new Webhooks(this);
 	this.recentItems = new RecentItems(this);
+	this.collaborationWhitelists = new CollaborationWhitelists(this);
```

Comment:
This should maybe be singular, since there's only one whitelist — similar to trash